docs/experience/integrations.md

```diff
@@ -98,6 +98,22 @@ descriptions: The Sematext Experience integrations cover all types of websites a
 			</div>
 		</a>
 	</div>
+	<div class="mdl-cell mdl-cell--4-col">
+		<a href="#vercel">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../../images/integrations/vercel.png" alt="Vercel" title="Vercel" style="padding-top:12px;height:145px;width:300px;">
+						</div>
+						<div class="side back">
+							<h5>Vercel</h5>Full support for all types of websites hosted on Vercel.
```

Comment:
Maybe Full support for ________________ for/or all types of ....

Reply:
Maybe Full support for "real user monitoring metrics" for/or all types of...

You mean like this?

Reply:
Something like that, yes.

Reply:
Added.